data-processing-lib/python/src/data_processing/utils/pipinstaller.py

```diff
@@ -0,0 +1,73 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import warnings
+warnings.filterwarnings('ignore')
+
+import subprocess
+import sys
+import pkg_resources
```

Comment:
the module `pkg_resources` is deprecated and will be removed, they recommend to use `[importlib.resources](https://docs.python.org/3.11/library/importlib.resources.html#module-importlib.resources)` instead

Reply:
I tried it, it does not seem to work

Reply:
you can do something like `subprocess.check_call([sys.executable, "-m", "pip", "show", "-q", name])`
and check return code